Gamer Event Page

MarilenaRoque opened the pull request:
##Gamer Event Page

I have developed a website for a fictitious Gamer Event, focusing on applying a little bit of all the things that I have learned in this month on the Microverse Program. :rocket: 

It contains a mobile and desktop version for three different pages:
- Home
- About
- Tickets

1. Navbar
Added Links
Sticky position
Responsive Dropdown Menu

2. Main
Differents backgrounds using image and gradient

3. Main Program 
Used box-shadow to highlight the events
Set up a row display for the desktop version

4. Championship
Added the more button to hide some items in the mobile version
Set up responsiveness

5. Responsive Sponsors Section

6. Footer
Links for my social medias
Event Icon

7. About
Set up a hover effect for the images and headers
Set up responsiveness

8. Tickets
Added form
Radio buttons for lunch and days choose
Sheets using grid
Different display style for different screen size

MarilenaRoque committed:
Added main section

Added Titles
Set up responsiveness
Added background

MarilenaRoque committed:
Added and Style Main Section

Added, styled and positioned the elements
Set up responsiveness
Added icons
Added and applied font style

MarilenaRoque committed:
Added and styled Events Section

Added all the Elements
Set up responsiveness
Set up hover effect for the h4 header
Styled box shadow for the events itens

MarilenaRoque committed:
Additional elements to the Events Section

Added button for mobile
Added see-all link for desktop

MarilenaRoque committed:
Added and Styled Championships Section

Added all the elements
Set up Responsiveness
Set up expansion effect with button "More"
Added and positioned images

MarilenaRoque committed:
Added and style Blue Navbar

Added Icons
Styled background color
Positioned elements

MarilenaRoque committed:
Added and Styled Sponsors Section

Added and positioned title and icon
Set up responsiveness

MarilenaRoque committed:
Added and styled Footer

Added the Links, icons and text
Positioned them
Set up resposiveness

MarilenaRoque committed:
About.html initial commit

Added initial structure
Added and style Main Section Elements
Set up responsiveness

MarilenaRoque committed:
Added and styled WebPage Developer Section

Added all the elements
Positioned all the elements
Added Icon link for the Github Account

MarilenaRoque committed:
Added and Styled Past Editions Section

Added all the elements
Set up background effect
Set up hover effect fo the Images

MarilenaRoque committed:
Added Sponsors Section for the About Page

MarilenaRoque committed:
Initial commit for Tickets.html page

Added the structure html
Added the Navbars
Added the Footer

MarilenaRoque committed:
Added links and Elements of the form

Added links to the navbar
Added form element to the ticket page

MarilenaRoque committed:
Fixing Image Path

MarilenaRoque committed:
Added links and Centralizing events itens

Added Navigation links for the other pages
Centralized the content of the Events Cards

MarilenaRoque committed:
Removing package nodule

MarilenaRoque committed:
Removing *.json

MarilenaRoque committed:
Added and styled Lunch Select Section

Added the form
Added Radio Buttons
Positioned Elements
Set Up responsiveness

MarilenaRoque committed:
Added Ticket Choice

Added and positioned inputs, labels and more elements
Set up border and table
Set up table responsiveness
Added confirm button

MarilenaRoque committed:
Added ticket page links

MarilenaRoque committed:
Sticklers and W3 code validation

MarilenaRoque committed:
Added Readme.md addicional information

MarilenaRoque committed:
Fixing Tickets Lable

MarilenaRoque committed:
Added demo video

MarilenaRoque committed:
Fixing Format

MarilenaRoque committed:
Fixing Font stylesheet format

MarilenaRoque committed:
Merge branch 'master' into develop

MarilenaRoque committed:
Removing unnecessary blank lines

MarilenaRoque committed:
Solving conflict

santiago-dev96 commented:
# Pull Request under review by @santiago-rodrig (TSE)

santiago-dev96 commented on index.html:
```diff
@@ -4,14 +4,293 @@
+   
 
     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="./assets/styles/reset.css" type="text/css">
```
Bootstrap already includes a reset
```suggestion
```

MarilenaRoque replied:
Got it. I will remove it before the merge. 
Thank you so much for your review. :smiley: :+1:

santiago-dev96 reviewed (approved):
# You have achieved competence :rocket:

Please be aware that capstone projects should be reviewed by several TSEs, you can keep track of the project from your dashboard, once there are no more reviews to be made you can proceed to merge this PR.

Congratulations for doing a great job.

MarilenaRoque committed:
Turn navbar opaque

MarilenaRoque committed:
Removed reset file

MarilenaRoque committed:
Added lists reset rule

MarilenaRoque committed:
Including semantic header

MarilenaRoque commented:
I have fixed all the requirements for this project

- [x] Set up an opaque navbar
- [x] The mobile and desktop version works perfectly
- [x] Added h1 header for semantic HTML

Thank you so much for the reviews! :smile: :rocket:

MarilenaRoque committed:
AddedGithub Live Demo
